### ISSUE TITLE
[tool] validator example base URLs buttons require context

### ISSUE THREAD

**Issue by UsmanAfzal** [WeAreSnook/OR-UK-frontend#199]
URL: https://validator.openreferraluk.org/

The try now buttons text should be replaced with the base URLs, the link text can then be removed. The paragraph under the heading "Example base URLs" should be updated to reflect this.

**Current**
Click on a try now button below to automatically populate the Base URL field.

<img width="702" alt="Screenshot 2021-07-05 at 16 15 52" src="https://user-images.githubusercontent.com/2459151/124492383-46fc8600-ddac-11eb-845a-b861194d1c2d.png">

**Change to**
Click on an example button below to automatically populate the Base URL field.

<img width="654" alt="Screenshot 2021-07-05 at 16 15 30" src="https://user-images.githubusercontent.com/2459151/124492338-39470080-ddac-11eb-9af1-d1e34e33f2de.png">

**Comment by UsmanAfzal** [comment]
@matt-porism I can see the update has been made to the buttons, the paragraph content as above is just left.

**Comment by matt-porism** [comment]
Thank you. I've removed the word "try now" from the paragraph.

**Comment by UsmanAfzal** [comment]
Checked and approved.
